Testscripts/Linux/utils.sh

```diff
@@ -2661,8 +2661,8 @@ function install_lagscope () {
 
 # Build and install ntttcp
 function build_ntttcp () {
-	ntttcp_version="v1.3.4"
-	# If the ntttcpVersion is provided in xml then it will go for that version, otherwise default to v1.3.4.
+	ntttcp_version="1.4.0"
```

Comment:
You put NTTTCP_VERSION parameter in ReplaceableTestParameters.xml. Why don't we use it here?

Reply:
if you are using 'ntttcp_version=NTTTCP_VERSION' like RDMA tests, it would be easier for your run-time. In RDMA/IB tests, we use lots of customized parameters.

Reply:
I knew, that is what we used in perf_ntttcp.sh, here build_ntttcp method used this version as default value when users **not specify** parameter for build_ntttcp, maybe no NTTTCP_VERSION in constants.sh at that time.

Reply:
Got it, so this is default value.